lib/update/cluster/phases/bootstrap.go

```diff
@@ -548,6 +576,49 @@ func (p *updatePhaseBootstrap) addUpdateRuntimePackageLabel() error {
 	return nil
 }
 
+// migrateInfluxDBData copies all from /var/lib/data/influxdb if it exist on the node and if destination
+// directory /var/lib/gravity/monitoring/influxdb empty
+func (p *updatePhaseBootstrap) migrateInfluxDBData(ctx context.Context) error {
+	newDataDirectory := "/var/lib/gravity/monitoring/influxdb"
+	oldDataDirectory := "/var/lib/data/influxdb"
+
+	if _, err := os.Stat(oldDataDirectory); os.IsNotExist(err) {
+		// influxdb data is on another node
+		return nil
+	}
+
+	empty, err := isEmptyDir(newDataDirectory)
+	if err != nil {
+		return trace.Wrap(err)
+	}
+	if !empty {
+		// directory already exist and not empty, probably data already migrated
+		return nil
+	}
+
+	p.Info("Copying InfluxDB data to the new directory.")
+	_, err = utils.RunCommand(ctx, p.FieldLogger,
+		utils.PlanetCommandArgs("cp", "-a", fmt.Sprintf("%s/.", oldDataDirectory), newDataDirectory)...)
```

Comment:
Does "newDataDirectory" need to be created first? Or at least check that it exists?